priv/static/specs/http_api_spec.yaml

```diff
@@ -79,3 +134,59 @@ definitions:
     properties:
       version:
         type: "string"
+  SessionsList:
+    type: "object"
+    properties:
+      TotalCount:
+        type: integer
+      Sessions:
+        type: array
+        items:
+          $ref: '#/definitions/SessionInfo'
+  SessionInfo:
+    type: "object"
+    properties:
+      IMSI:
+        type: string
+      IMEI:
+        type: string
+      MSIDN:
```

Comment:
should be MSISDN

Reply:
oh, will fix this and in other places too